install.sh

```diff
@@ -11,6 +11,6 @@ composer require guzzlehttp/guzzle:^6.0
 composer dump-autoload
 
 cd plugins/BillTech
-chmod 0644 cron/*
+chmod 0755 cron/*
```

Comment:
A po co? Tam nie ma plików wykonywalnych.

Reply:
Często występuje problem z brakiem uprawnień do wykonywania jak wtyczka zostanie pobrana jako zip z Github.

Reply:
Ale pliki w tym katalogu nie powinny być wykonywane. Musi chodzić o coś innego.

Reply:
Ok, przypomniałem sobie dokładniej ten temat. Chodzi o pliki w folderze bin tj.:

- billtech-update-links.php
- billtech-update-payments.php
- timestamp.sh

Jak nie nadamy uprawnień do wykonywania, to leci permission denied. Zaraz poprawię.